x-pack/plugins/lens/public/shared_components/coloring/color_ranges/utils/color_ranges_extra_actions.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getDataMinMax, roundValue } from '../../utils';
+
+import type { ColorRange, DataBounds } from '../types';
+import type { CustomPaletteParamsConfig } from '../../../../../common';
+import type { PaletteContinuity } from '../../../../../../../../src/plugins/charts/common';
+
+/**
+ * Distribute equally
+ * @internal
+ */
+export const distributeEqually = (
+  colorRanges: ColorRange[],
+  rangeType: CustomPaletteParamsConfig['rangeType'],
+  continuity: PaletteContinuity,
+  dataBounds: DataBounds
+) => {
+  const items = colorRanges.length;
+  const lastIndex = colorRanges.length - 1;
+  const { min, max } = getDataMinMax(rangeType, dataBounds);
+  const step = roundValue((max - min) / items, 2);
+
+  const getValueForIndex = (index: number) => roundValue(min + (step * 100 * index) / 100);
```

Comment:
Can you remove the `* 100 / 100` here?
`roundValue` already provides this type of rounding.